### DISCUSSION TITLE
Retrieve the last (x) inserted entities

### DISCUSSION THREAD

**Discussion by slayermm1122** [milvus-io/milvus#23419] [Q&A and General discussion] [answered]
What is fastest way to retreive, say last 5 inserted entities? Do we need create an schema to record inserting time or there is a much easier way to get this? Does milvus have any embedded hidden inserting time value for each entities?

Thanks

**Comment by xiaofan-luan** [comment, marked as answer]
> What is fastest way to retreive, say last 5 inserted entities? Do we need create an schema to record inserting time or there is a much easier way to get this? Does milvus have any embedded hidden inserting time value for each entities?
> 
> Thanks

Thank you for reaching out.

Have you considered looking into Change Data Capture? If you only do it occasionally, then maybe has another user timestamp field and query with timestamp range?

**Reply by moonSandra** [reply]
> Thank you for reaching out.
> 
> Have you considered looking into Change Data Capture? If you only do it occasionally, then maybe has another user timestamp field and query with timestamp range?

Hi @xiaofan-luan 
how can we do this? i want to retrieve the last entity which inserted to milvus. 
should i run a 'while True' code to find it ?!

**Reply by xiaofan-luan** [reply]
> !

So you want to query orderby timestmp limit 100 , is that the request?
